packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -144,6 +144,24 @@ const watchKonnectorAccount = (client, account, options) => {
   return accountWatcher
 }
 
+/**
+ * Delete an account
+ * @param  {Object}  client  CozyClient
+ * @param  {Object}  account io.cozy.accounts document
+ */
+const deleteAccount = async (client, account) => {
+  try {
+    await client.destroy(account)
+  } catch (error) {
+    if (error.status === 409) {
+      const syncedAccount = await findAccount(client, account._id)
+      await client.destroy(syncedAccount)
```

Comment:
We should add a try / catch block here?

Reply:
It will be [caught by the component itself](https://github.com/cozy/cozy-libs/pull/451/files#diff-44a2d2347aa83b5d53cc4ea6e2981d5eR26) to run `onError` props, it should be enough or I missed something?

Reply:
Should be enough. Thanks

Reply:
This behavior could be in cozy-client IMHO.

```
client.destroy(account, { force: true })
```